driver/node.go

```diff
@@ -337,7 +348,21 @@ func (d *Driver) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandVolume
 	d.log.WithField("method", "node_expand_volume").
 		Info("node expand volume called")
 
-	return nil, status.Error(codes.Unimplemented, "")
+	volumeID := req.GetVolumeId()
+	if len(volumeID) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Volume ID not provided")
+	}
+
+	r := resizefs.NewResizeFs(&mount.SafeFormatAndMount{
+		Interface: mount.New(""),
+		Exec:      mount.NewOsExec(),
+	})
+	// todo: find device path
```

Comment:
@adamwg - I wasn't sure how to find the device path needed to resize: https://godoc.org/k8s.io/kubernetes/pkg/util/resizefs
Can you please help me out?

Reply:
Doesn't look like we get the volume name or device path in the request, so we'll probably have to infer it from the mount path. Looks like [`mount.GetDeviceNameFromMount`](https://godoc.org/k8s.io/kubernetes/pkg/util/mount#GetDeviceNameFromMount) might be helpful here.